src/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The three files in src and .settings should not be committed. Check your .gitignore in the main folder

Reply:
noted